person.rb

```diff
@@ -1,9 +1,11 @@
 require_relative 'nameable'
 require_relative './decorators/capitalize'
 require_relative './decorators/trim'
+require_relative 'book'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
[ Optional Changes ]
The previous reviewer did not spot this but I highly recommend you make the change 👍🏽

- [ ] Getters and Setters for @rentals should be present. Kindly modify to `attr_accessor :name, :age, :rentals`

Reply:
Thanks for the information. I will also create the seet for `rentals`